Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/TooltipEnums/ConnectorPivotDirectionType.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.ToolTips
+{
+    /// <summary>
+    /// In which direction does the tooltip connector project.
+    /// </summary>
+    public enum ConnectorPivotDirectionType
+    {
+        Manual = 0,         // Direction will be specified manually
+        North,
```

Comment:
Which way is "North"? 😅 

I wonder if we can come up with a better direction indicator based on X,Y,Z directions.

Reply:
North: Vector3.up
NorthEast: Lerp(V3.up, V3.right, .5f).normalized
East: Vector3.right
etc

Should I replace the NESW naming approach to Up/Right/Down/Left with UpRight, DownRight, DownLeft, UpLeft?

Reply:
I think we might already have an enum that has something similar? Not really sure.

I think this might be a good one to make generic.

Reply:
Should I generalize and migrate to MRTK.Core, probably into the Utilities?

I'll look around to see if anything already exists.

Reply:
Most of our enums are in "Core/Definitions"

Reply:
Maybe we should take the lead from Vector3.Up, Vector3.Forward, Vector3.Left, Vector3.Back etc?
https://docs.unity3d.com/ScriptReference/Vector3.html

Up
UpRight
UpLeft
UpForwad
UpBack

Down
etc

?

Reply:
Next is the question of should I only include the options that this enum is used for, or should I add the options that weren't previously used 'ForwardRight' for instance.
Some part of me questions if it is really better to have this as an enum compared to a Vector3 if it's being used exactly like a Vector3 but with greater long term maintenance requirements.